### PR TITLE
feat: update nx 15.6.3 and Angular 15.1

### PR DESCRIPTION
**Pull request by fpaul-1A**

### CODE REVIEW

**Review by matthieu-crouzet — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

File: apps/vscode-extension/project.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
             "input": ".",
             "output": "."
           }
-        ]
+        ],
+        "updateBuildableProjectDepsInPackageJson": false
```
Comment (vscaiceanu-1a) [inline apps/vscode-extension/project.json:29]: why is this necessary?

**Review by vscaiceanu-1a — DISMISSED** [review]

**Comment by kpanot** [comment]
- [ ] should clean peerDeps
image

File: apps/vscode-extension/project.json (hunk 1 above)
Comment (fpaul-1A) [reply apps/vscode-extension/project.json:29]: `@nrwl/js:tsc` executor is failing with error `Cannot find dist/package.json` if I don't disable it, I suspect the package.json are not copied to the dist yet when running this

**Review by kpanot — DISMISSED** [review]

**Review by kpanot — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]
